tests/testthat/test-model-summary.R

```diff
@@ -105,6 +105,7 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
 
       for (.d in names(ref_sum$run_details)) {
         if (.d != "output_files_used") {
+          skip_if_old_bbi('3.1.1')
```

Comment:
This skip condition isn't changing within the loop, so I think this could be moved above line 106.